pkg/rancher-desktop/components/Images.vue

```diff
@@ -190,36 +190,36 @@ export default {
         .map(this.getTaggedImage);
     },
     rows() {
+      const protectedImages = this.protectedImages;
```

Comment:
It doesn't look like this temporary is necessary. Actually, it looks like only the `if (!image.availableActions)` removal is necessary?

I guess that makes sense, because we used to never update the actions list…

Reply:
Hmm. I am observing the `rows` computed prop triggering without. Perhaps, my initial assessment was incorrect 🤔